index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    <title>Resume</title>
+    <link href="/assets/css/main.css" rel="stylesheet" type="text/css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Merriweather|Oswald&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="flex-container-column" id="container">
+      <header id="header">
+        <div class="flex-container-row">
+          <div id="image"></div>
+          <div>
+            <h2 id="name"></h2>
+            <h3 id="label"></h3>
+            <p id="contact">
+              <span id="email"></span>
+              <span id="phone"></span>
+              <span id="website"></span>
+              <span id="address"></span>
+            </p>
+            <p id="profiles"></p>
+          </div>
+        </div>
+      </header>
+      <div id="summary"></div>
+      <div id="skills">
+        <h3>Skills</h3>
+      </div>
+      <div id="jobs">
+        <h3>Employment History</h3>
+      </div>
+      <div id="volunteer">
+        <h3>Volunteer Experience</h3>
+      </div>
+      <div id="education">
+        <h3>Education</h3>
+      </div>
+      <div id="awards">
+        <h3>Awards</h3>
+      </div>
+      <div id="publications">
+        <h3>Publications</h3>
+      </div>
+      <div id="languages">
+        <h3>Languages</h3>
+      </div>
+      <div id="interests">
+        <h3>Interests</h3>
+      </div>
+      <div id="references">
+        <h3>References</h3>
+      </div>
```

Comment:
In the short time frame we had, I think your initial structure is well put together.  Only thing I would suggest at this point, is maybe using more semantical HTML rather than all the divs (I see you used header which is good, for the other ones you could also use section or article), and actually have wrapped this in a main tag.  Otherwise good stuff!!